### PR TITLE
UICIRCLOG-57 Removing "loan details" option for log entries containing multiple barcodes

### PR DESCRIPTION
**Pull request by uladislausamets**
## Purpose

https://issues.folio.org/browse/UICIRCLOG-57
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/CirculationLogList/CirculationLogEventActions/CirculationLogEventActions.test.js [hunk 1]
```diff
@@ -44,18 +46,18 @@ jest.mock('@folio/stripes/components', () => {
...
   getHasNoticePolicyDetails: jest.fn(),
 }));
 
-const renderCirculationLogEventActions = ({ item, referenceIds } = {}) => (render(
+const renderCirculationLogEventActions = ({ object, items, referenceIds } = {}) => (render(
   <CirculationLogEventActions
-    item={item}
+    items={items}
     referenceIds={referenceIds}
+    object={object}
```
Comment (aliaksei-chumakou) [inline src/CirculationLogList/CirculationLogEventActions/CirculationLogEventActions.test.js:60]: it's quite unclear what object is meaning

File: src/CirculationLogList/CirculationLogEventActions/CirculationLogEventActions.js [hunk 2]
```diff
@@ -164,11 +163,11 @@ export const CirculationLogEventActions = ({ object, item, referenceIds }) => {
...
 CirculationLogEventActions.propTypes = {
   object: PropTypes.string,
-  item: PropTypes.object,
+  items: PropTypes.arrayOf(PropTypes.object),
   referenceIds: PropTypes.object,
 };
 
 CirculationLogEventActions.defaultProps = {
-  item: {},
+  items: [{}],
```
Comment (aliaksei-chumakou) [inline src/CirculationLogList/CirculationLogEventActions/CirculationLogEventActions.js:171]: imho, dangerous - empty array can be provided

File: src/CirculationLogList/CirculationLogEventActions/CirculationLogEventActions.js [hunk 3]
```diff
@@ -21,15 +21,14 @@ import {
...
 } from './utils';
 
-export const CirculationLogEventActions = ({ object, item, referenceIds }) => {
+export const CirculationLogEventActions = ({ object, items, referenceIds }) => {
   const intl = useIntl();
   const stripes = useStripes();
-
   const { feeFineId, userId, requestId, noticePolicyId, templateId } = referenceIds;
-  const { loanId } = item;
+  const { loanId } = items[0];
```
Comment (axelhunn) [inline src/CirculationLogList/CirculationLogEventActions/CirculationLogEventActions.js:28]: image

**Review by axelhunn — APPROVED** [review]

File: src/CirculationLogList/CirculationLogEventActions/CirculationLogEventActions.test.js [hunk 4]
```diff
@@ -77,34 +78,43 @@ describe('Given Circulation Log Event Actions', () => {
     stripes.hasPerm.mockReset();
   });
 
-  it('Than it should not render Dropdown actions when no actions available', () => {
+  it('Then it should not render Dropdown actions when no actions available', () => {
```
Comment (aliaksei-chumakou) [inline src/CirculationLogList/CirculationLogEventActions/CirculationLogEventActions.test.js:81]: it's hard to read, maybe just skip `Then it` ? it'll look like `it('should not render...` and makes more sense, IMHO

**Review by aliaksei-chumakou — APPROVED** [review]

File: src/CirculationLogList/CirculationLogEventActions/CirculationLogEventActions.test.js (hunk 4 above)
Comment (axelhunn) [reply src/CirculationLogList/CirculationLogEventActions/CirculationLogEventActions.test.js:81]: I would also get rid of "should"
```js
describe("A suite", function() {
  it("contains spec with an expectation", function() {
    expect(true).toBe(true);
  });
});
```
from https://jasmine.github.io/2.0/introduction

Comment (axelhunn) [reply src/CirculationLogList/CirculationLogEventActions/CirculationLogEventActions.test.js:81]: Just let's do it in another PR

**Pull request merged**